### PR TITLE
Mp user profile

### PR DESCRIPTION
**Pull request by Ritumka**
I'm working on user profile.

### CODE REVIEW

File: app/views/users/show.html.erb
Comment (jessjno) [inline app/views/users/show.html.erb:1]: nice formatting

**Review by jessjno — APPROVED** [review]
looks good

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully created." }
+        format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
```
Comment (marywebby) [inline app/controllers/comments_controller.rb:29]: Good job implementing `redirect_back`!

File: app/views/comments/_form.html.erb [hunk 2]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :photo_id, style: "display: block" %>
-    <%= form.text_field :photo_id %>
-  </div>
+  <%= form.hidden_field :photo_id %>
 
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <div class="mb-3">
```
Comment (marywebby) [inline app/views/comments/_form.html.erb:16]: Looks good!

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  <div class="row mb-4">
+    <div class="col-md-6 offset-md-3">
+      <div class="card">
+        <%= image_tag photo.image, class: "card-img-top" %>
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
```
Comment (marywebby) [inline app/views/users/show.html.erb:17]: nice job adding some styling to the comments here!

**Review by marywebby — APPROVED** [review]
Nice job on this Margo!

**Pull request merged**